spec/controllers/users_controller_spec.rb

```diff
@@ -1,88 +1,84 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe Clearance::UsersController do
   it { should be_a Clearance::BaseController }
 
-  describe 'when signed out' do
-    before { sign_out }
+  describe "on GET to #new" do
+    context "when signed out" do
+      context "with email" do
```

Comment:
I'd get rid of this context and change the order of these two it blocks to say something like:

```
it "renders a form for a new user"
it "renders a form for a user with the query-string-provided email"
```

As a general rule don't use "should" in the it block description.